spec/unit/notifications.spec.ts

```diff
@@ -134,7 +134,7 @@ describe("fixNotificationCountOnDecryption", () => {
 
         fixNotificationCountOnDecryption(mockClient, event);
 
-        expect(room.getUnreadNotificationCount(NotificationCountType.Total)).toBe(2);
+        expect(room.getUnreadNotificationCount(NotificationCountType.Total)).toBe(3);
```

Comment:
I think this is meant to be 3. The test sets up an event with a `getPushActions()` value of `{notify: false}` but `getPushActionsForEvent` returns `{notify: true}`. Ergo, I think this is meant to increase the room's total count.